subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ArtifactoryAndDockerSmokeTest.groovy

```diff
@@ -125,6 +122,11 @@ class ArtifactoryAndDockerSmokeTest extends AbstractPluginValidatingSmokeTest {
                     "This is scheduled to be removed in Gradle 9.0. " +
                     "Consult the upgrading guide for further information: https://docs.gradle.org/${GradleVersion.current().version}/userguide/upgrading_version_7.html#org_gradle_util_reports_deprecations"
             )
+            runner.expectLegacyDeprecationWarning(
```

Comment:
This (getting the same warning with different upgrade guide links) feels weird to me... I understand that older Gradle versions produced deprecation warnings pointing to the v7 upgrade guide, but why keep both in this Gradle version? Why not update all deprecations produced by this Gradle version to point to the v8 upgrade guide? DISCLAIMER: I'm not sure what I'm suggesting is correct and that I'm not missing some aspect of it. Would be good to get another opinion.

Reply:
If the messages were different, I would be OK with two different deprecation warnings.

But since the message is the same (and that's fine, no need to change it), I believe we should have a single message indeed.
So I would suggest to update the whole `ConfigureUtil` deprecation setup to report a link to Gradle 8 migration docs.

However, we should also update the upgrade guide entry, to make sure it covers this wider scope.